compiler/compiler.go

```diff
@@ -1655,6 +1655,7 @@ func (b *builder) createFunctionCall(instr *ssa.CallCommon) (llvm.Value, error)
 		// Direct function call, either to a named or anonymous (directly
 		// applied) function call. If it is anonymous, it may be a closure.
 		name := fn.RelString(nil)
+		name = strings.Replace(name, "tinygo.org/x/", "", 1)
```

Comment:
Please update the names below (like `device/arm.SVCall`) instead of using a string replace.